### ISSUE TITLE
Publish new version

### ISSUE THREAD

**Issue by zummenix** [RustAudio/lewton#15]
Could you please publish new version with 09857bfc4db0c8494aa5cf6b269c6a3018da3659?

**Comment by est31** [comment]
Do you want to be able to compile with stable, or is the ability to compile with beta and nightly enough?

Right now, the master version of lewton requires Rust 1.18 which means you can't compile it with stable, but I could backport the patch if ability to compile on stable is important to you.

**Comment by est31** [comment]
Also note that right now you can easily work around the build failure of the ieee754 crate, just do in your Cargo.toml:

```toml
[dependencies]
lewton = { version = "0.5", default-features = false, features = ["ogg"] }
```

**Comment by zummenix** [comment]
Never mind, I was trying to build https://github.com/tomaka/rodio using nightly rust. With stable it builds fine.

**Comment by est31** [comment]
Okay, I have published 0.5.1 now, which includes the commit and a few other bugfixes.
